### PR TITLE
updated readme file

### PR DESCRIPTION
**Pull request by markhaur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+## Maintainers
+[Jarri Abidi](https://github.com/jarri-abidi)\
+[Faisal Nisar](https://github.com/markhaur)
+
+## Credits
+This project exists, thanks to all the people who contribute.
+
+<a href="https://github.com/jarri-abidi/todo/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=jarri-abidi/todo" />
+</a>
```
Comment (jarri-abidi) [inline README.md]: I think I'll write something about contributing guidelines myself when I'm ready. For now let's remove this.
Also, I don't want this project to be about contributors. Only the code is important.
```suggestion
```
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
 # todo
-Example project meant to demonstrate a production-ready, enterprise-style REST API using go-kit. 
+[![MIT](https://img.shields.io/github/license/jarri-abidi/todo)](https://github.com/jarri-abidi/todo/blob/master/LICENSE)
 
-To run:
+**todo** is an example application written in [Go](https://go.dev/) meant to demonstrate a production-ready, enterprise-style REST API using go-kit.
```
Comment (jarri-abidi) [inline README.md]: could we add a link to the go-kit.io website too?

Comment (markhaur) [reply README.md]: yeah we can do that... lemme do it

**Review by jarri-abidi — COMMENTED** [review]
Thank you @markhaur

**Review by jarri-abidi — APPROVED** [review]

**Pull request merged**